handler/permissions_handler.go

```diff
@@ -128,27 +128,35 @@ func GetRepoConfig(owner string, repository string) (*types.DerekRepoConfig, err
 }
 
 func buildDerekConfig(client http.Client, bytesConfig []byte) (*types.DerekRepoConfig, error) {
```

Comment:
How many redirects do we allow?

Reply:
How are we preventing cyclic redirects?

Reply:
It only follows the first url's redirect.